docs/monitoring/containers.md

```diff
@@ -1,35 +1,262 @@
+- Unmount – when a volume is removed from a container
+
+### Container network events
+
+- Create – when a  network is created
+- Connect – when a container connects to a network
+- Remove – when the network is removed
+- Destroy – when a network is destroyed
+- Disconnect – when a container disconnects from a network
+
+### Container daemon events
+
+- Reload
+
+### Container services, nodes, secrets, and config events
+
+- Create – on the creation of a resource
+- Remove – on the removal of a resource
+- Update – on the creation of a resource
+
+![](../images/integrations/docker/events.png)
+
+## Container Metrics Overview
+
+The following information is collected and transmitted to Sematext Cloud or Sematext Enterprise.
+
+<table>
+<tbody>
+<tr class="odd">
+<td><strong>Type</strong></td>
+<td><strong>Description</strong></td>
+</tr>
+<tr class="even">
+<td><strong>Operating System Metrics</strong></td>
+<td><p>Host machine metrics</p>
+<ul>
+<li>CPU Usage</li>
+<li>Memory Usage</li>
+<li>Network Stats</li>
+<li>Disk I/O Stats</li>
+</ul></td>
+</tr>
+<tr class="odd">
+<td><strong>Container Metrics/Stats</strong></td>
+<td><ul>
+<li>CPU Usage / limits</li>
+<li>Memory Usage / Limits / Fail Counters</li>
+<li>Network Stats</li>
+<li>Disk I/O Stats</li>
+</ul></td>
+</tr>
+<tr class="even">
+<td><strong>Events</strong></td>
+<td> </td>
+</tr>
+<tr class="odd">
+<td> Agent Startup Event</td>
+<td>server-info – created by spm-agent framework with node.js and OS version info on startup. Please note the agent is implemented in node.js.</td>
+</tr>
+<tr class="even">
+<td> </td>
+<td>Docker-info – Docker Version, API Version, Kernel Version on startup</td>
+</tr>
+<tr class="odd">
+<td>Docker Events</td>
+<td>Container Lifecycle Events| create, exec_create, destroy, export, ...</td>
+</tr>
+<tr class="even">
+<td>Container Runtime Events</td>
+<td>die, exec_start, kill, pause, restart, start, stop, unpause, ...</td>
+</tr>
+<tr class="odd">
+<td><strong>Docker Logs</strong></td>
+<td> </td>
+</tr>
+<tr class="even">
+<td>Default Fields</td>
+<td><ul>
+<li>hostname / IP address</li>
+<li>container id</li>
+<li>container name</li>
+<li>image name</li>
+<li>message</li>
+</ul></td>
+</tr>
+<tr class="odd">
+<td><p>Log formats</p>
+<p>(detection and log parsers)</p></td>
+<td><ul>
+<li>NGINX</li>
+<li>APACHE httpd, Kafka, Solr, HBase, Zookeeper, Cassandra</li>
+<li>MySQL</li>
+<li>MongoDB</li>
+<li>Redis</li>
+<li>Elasticsearch</li>
+<li>NSQ  / <a href="http://Nsq.io" class="external-link">Nsq.io</a></li>
+<li>patterns are maintained here: 
+<ul>
+<li><a href="https://github.com/sematext/logagent-js" class="uri" class="external-link">https://github.com/sematext/logagent-js</a></li>
+</ul></li>
+</ul></td>
+</tr>
+<tr class="even">
+<td> </td>
+<td><span>JSON, Plain Text</span></td>
+</tr>
+</tbody>
+</table>
+
+**Supported Platforms**
+
+  - Docker Engine \>= 17.0.0
+  - Platforms using Docker:  
+      - Docker Cloud
+      - Docker Data Center
```

Comment:
Is this Docker Enterprise now?

Reply:
Not sure, but I think not.

Reply:
OK.  We should then add Docker Enterprise here.